docs/submitting-molecular-data.md

```diff
@@ -2,11 +2,86 @@
 id: submitting-molecular-data
 title: Submitting Molecular Data
 ---
+Molecular data will be submitted to your local Regional Data Processing Centre (RDPC). RDPCs are responsible for processing your programs molecular data according to the [Harmonization Pipeline](). If you are unsure what the correct RDPC is to submit to, please [contact the DCC](https://platform-ui.qa.argo.cancercollaboratory.org/contact).
 
-## Installing the Song and Score Clients
 
-## Formatting Molecular Metadata Payloads
+This guide will describe how to submit molecular data to the ARGO Data Platform. Molecular data consists of all raw data  files generated from your program's donors, as well as any associated file metadata.
 
-## Uploading Molecular Files
+## Data Submission Client Configuration
+Molecular data is uploaded to the ARGO Data Platform using the Song and Score CLIs (Command Line Clients).  Song is an open source system used to track and validate metadata about raw data submissions. Score securely and quickly manages upload and download of files to cloud repositories. The Song and Score clients are used in conjuction to  
 
-## Publishing Molecular Data for Processing
+### Song-Client
+Download the **[latest version of the Song client](https://artifacts.oicr.on.ca/artifactory/dcc-release/bio/overture/song-client/4.0.0/song-client-4.0.0-dist.tar.gz)**. Once you have unzipped the tarball, update the  `/conf/application.yaml` configuration file with the correct user and data submission program values, including:
+- local RDPC Song server URL
+- [API Key](user-profile-and-token)
+- ARGO Program Id that you are submitting to
```

Comment:
suggestion (since we use studyId in the file)

ARGO Program Id that you are submitting to (called studyId within the file)